.gitattributes

```diff
@@ -1,3 +1,3 @@
 *.ipynb filter=nbstripout
 *.ipynb diff=ipynb
-stglib/_version.py export-subst
+stglib/_version.py export-subst
```

Comment:
Please ensure you aren't modifying the root .gitattributes file so we don't unnecessarily altering its git history. (it looks like you  mayneed a newline at the end of the file. That's why there is the red circle on the github diff view).

Reply:
Added new line.